Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -1209,6 +1234,57 @@ instance : IsTotallyReal ℚ where
     rw [Subsingleton.elim v Rat.infinitePlace]
     exact Rat.isReal_infinitePlace
 
+section maximalRealSubfield
+
+open ComplexEmbedding
+
+/--
+The maximal real subfield of `K`. It is totally real, see `isTotallyReal_maximalRealSubfield`, and
```

Comment:
Can you add the corresponding `doc`  at the top of the file?

Reply:
Not sure to understand what you mean...

Reply:
The file `NumberField.Embeddinds` has been split in the meantime.

Reply:
OK, so ping me again once this is done!

Reply:
@faenuccio Should be okay now